src/bandersnatch_filter_plugins/allowlist_name.py

```diff
@@ -121,7 +121,7 @@ def _parse_package_lines(package_lines: List[str]) -> Set[Requirement]:
     filtered_requirements: Set[Requirement] = set()
     for package_line in package_lines:
         package_line = package_line.strip()
-        if not package_line or package_line.startswith("#"):
+        if not package_line or package_line.startswith(("#", "-"))):
```

Comment:
```suggestion
        if not package_line or package_line.startswith(("#", "-")):
```
Seems editing python via the web is evil / I am useless :P